shanoir-ng-datasets/src/main/java/org/shanoir/ng/examination/controler/ExaminationApiController.java

```diff
@@ -85,10 +76,13 @@ public class ExaminationApiController implements ExaminationApi {
 	private CenterRepository centerRepository;
 
 	@Autowired
-	ShanoirEventService eventService;
+	private ShanoirEventService eventService;
+
+	@Autowired
+	private StudyRepository studyRepository;
 
 	@Autowired
-	StudyRepository studyRepository;
+	private StudyRightsService rightsService;
```

Comment:
It seems unused.

If not, you will have to mock it in the failing unit test (like the other @Autowired dependencies) : 

```
2023-05-04T15:36:21.3596446Z Tests in error: 
2023-05-04T15:36:21.3600774Z   ExaminationApiControllerTest.deleteExaminationTest » IllegalState Failed to lo...
2023-05-04T15:36:21.3601628Z   ExaminationApiControllerTest.findExaminationByIdTest » IllegalState Failed to ...
2023-05-04T15:36:21.3602328Z   ExaminationApiControllerTest.findExaminationsTest » IllegalState Failed to loa...
2023-05-04T15:36:21.3603700Z   ExaminationApiControllerTest.saveNewExaminationTest » IllegalState Failed to l...
2023-05-04T15:36:21.3604428Z   ExaminationApiControllerTest.testAddExtraData » IllegalState Failed to load Ap...
2023-05-04T15:36:21.3605533Z   ExaminationApiControllerTest.testDeleteExaminationWithExtraData » IllegalState
2023-05-04T15:36:21.3606563Z   ExaminationApiControllerTest.testDownloadExtraData » IllegalState Failed to lo...
2023-05-04T15:36:21.3607347Z   ExaminationApiControllerTest.testDownloadExtraDataNotExisting » IllegalState F...
2023-05-04T15:36:21.3608034Z   ExaminationApiControllerTest.updateExaminationTest » IllegalState Failed to lo...
```
```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'examinationApiController': Unsatisfied dependency expressed through field 'rightsService'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.shanoir.ng.study.rights.StudyRightsService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
```

Reply:
@michaelkain @julien-louis tests are all right now, thx !